scripts/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Same here, builds are already handled in make files, no need to duplicate it in script files.

Reply:
sounds good. this is in place as a temporary usage. Feel free to delete as part of a cleanup before merging to main